content/concepts/introduction/intro-to-consensus.md

```diff
@@ -0,0 +1,63 @@
+# Introduction to Consensus
+
+_Consensus_ is the most important property of any decentralized payment system. In traditional centralized payment systems, one authoritative administrator gets the final say in how and when payments occur. Decentralized systems, by definition, don't have an operator to do that. Instead, decentralized systems like the XRP Ledger define a set of rules all participants follow, so every participant can agree on the exact same series of events and their outcome at any point in time. We call this set of rules a _consensus protocol_.
```

Comment:
A nit, but perhaps instead of "operator," we can continue to use "administrator" from the previous sentence? Just to make the point even clearer.

Reply:
I really like the way you expressed the heart of consensus as agreement "on the exact same series of events and their outcome at any point in time." I think that sometimes when people think about what financial systems do -- it all seems much more complicated! But when it comes down to it, this is what it is, and I think it is great to lead off the article with this concept which is quite grok-able.